types/errors.go

```
@@ -0,0 +1,33 @@
+package types
+
+import "errors"
+
+var (
+	// Chain initialization errors
```

golangci-lint (1.62.2)

[warning] exported: comment on exported var ErrZeroInitialHeight should be of the form "ErrZeroInitialHeight ..."

(revive)